### PR TITLE
Changed fmt.Println to loggo. Fix #140

### LINKED ISSUE #140: Make sure we use logging and not fmt.Print(ln) in the code

### PR DESCRIPTION
**Pull request by martinpinto**
Changed all fmt.Println references to use njpatel/loggo

### CODE REVIEW

File: src/skizze-cli/bridge/loop.go [hunk 1]
```diff
@@ -125,20 +128,19 @@ func Run() {
...
-			fmt.Println("")
 			line.AppendHistory(query)
 		} else if err == liner.ErrPromptAborted {
-			log.Print("Aborted")
+			log.Printf("Aborted")
 			tearDownClient(conn)
 			return
 		} else {
-			log.Print("Error reading line: ", err)
+			log.Fatalf("Error reading line: %s", err.Error())
```
Comment (njpatel) [inline src/skizze-cli/bridge/loop.go]: I don't think `Fatalf` is needed here, it'll call os.Exit(), which won't be needed for user error (or even writing the history)

File: src/server/server.go [hunk 2]
```diff
@@ -62,7 +62,7 @@ func unmarshalDom(e *storage.Entry) *pb.Domain {
 }
 
 func (server *serverStruct) replay() {
-	fmt.Println("Replaying ...")
+	log.Printf("Replaying ...")
```
Comment (seiflotfy) [inline src/server/server.go]: Why are you mixing logger and log?

Comment (martinpinto) [inline src/server/server.go]: I've coded the following conventions:
- All user input errors in cli are now being handled by “log” (e.g. invalid operation, which show a timestamp)
- All responses from cli are being handled by “fmt” (e.g. response from GET CARD, no timestamp)
- All program errors are being handled by “njpatel/loggo" (more elaborate debug logging)

Comment (seiflotfy) [inline src/server/server.go]: in this case this is still wrong....
This is replaying on theprogram side. it should be loggo

File: src/skizze/main.go [hunk 3]
```diff
@@ -32,8 +31,8 @@ func main() {
 	err := os.Setenv("SKIZZE_PORT", strconv.Itoa(int(port)))
 	utils.PanicOnError(err)
 
-	logger.Infof("Starting Skizze...")
-	logger.Infof("Using data dir: %s", config.GetConfig().DataDir)
+	log.Printf("Starting Skizze...")
```
Comment (seiflotfy) [inline src/skizze/main.go]: Same here
Everything in src/skizze should be done using loggo

**Pull request merged**